recipes/daetk/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
```

Comment:
This line is not necessary, could you please remove it? It may cause some problems
``conda-build`` already handle the shebangs
```suggestion
```